src/Umbraco.Web.UI.Client/src/packages/core/workspace/entity-detail/entity-detail-workspace-base.ts

```diff
@@ -393,7 +393,7 @@ export abstract class UmbEntityDetailWorkspaceContextBase<
 	 * Check if there are unpersisted changes.
 	 * @returns { boolean } true if there are unpersisted changes.
 	 */
-	protected _getHasUnpersistedChanges(): boolean {
+	public getHasUnpersistedChanges(): boolean {
```

Comment:
To avoid introducing breaking changes for anyone extending the `EntityDetailWorkspaceBase`, I believe the best approach is to create a new public method that calls the existing protected method. This way, we can deprecate the protected method and remove it in a future major version.